.azure-pipelines/code-gen.yml

```diff
@@ -21,7 +21,8 @@ jobs:
   pool: pool-windows-2019
   steps:
   - checkout: self
-  - template: util/get-github-pat-steps.yml
+    persistCredentials: true
+  - template: util/get-github-token-steps.yml
```

Comment:
Do we still use `get-github-pat-steps.yml` anywhere? If not, can we delete it?

Reply:
It's still used in several pipelines. Can't delete it right now. The next step is to replace it with `get-github-token-steps.yml` in other pipelines (internal or release pipelines). Let's wait for a few days to see if public pipelines are all good.